### ISSUE TITLE
don't pass tile list via arguments

### ISSUE THREAD

**Issue by 174n** [derhuerst/combine-tiles#1]
I have an object with >1000 tiles and when I'm trying to combine them I get this error:
```
internal/child_process.js:358
    throw errnoException(err, 'spawn');
    ^

Error: spawn ENAMETOOLONG
    at ChildProcess.spawn (internal/child_process.js:358:11)
    at spawn (child_process.js:538:9)
    at Object.execFile (child_process.js:216:15)
    at exec (child_process.js:147:18)
    at combineTiles (C:\Users\...\node_modules\combine-tiles\index.js:33:2)
    at Object.<anonymous> (C:\....js:29:1)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
```
I know why it occurs but I don't know how to fix it.
I don't think I can achieve this without rewriting the whole thing without using commands to combine tiles

**Comment by derhuerst** [comment]
Can you give more information about the environment you're trying to run this in? e.g. operating system, file system, etc.

After googling, I assume it's related to the file system. https://serverfault.com/questions/9546/filename-length-limits-on-linux/9548#9548

**Comment by 174n** [comment]
I used windows 10 ltsb.
I thisk it's not because of the filesystem. All the filenames was short.
If I understood it right the tool generates a gm command with all filenames and other parameters. If this command gets too long I encounter that error

**Comment by derhuerst** [comment]
Not sure how we could make the command shorter. Any ideas?

**Comment by 174n** [comment]
I don't think you can make it work with the command execution. You probably need to use some native js library to process images (like jimp or sharp). You also get Buffer support with them

**Comment by derhuerst** [comment]
> I don't think you can make it work with the command execution.

Why do you think so? There are lots of other npm libs that spawn subprocesses and AFAIK work on Windows.

**Comment by derhuerst** [comment]
[Currently the list of files is passed via arguments](https://github.com/derhuerst/combine-tiles/blob/13d1c7818658cfe4cd307fb5def0e055f56ba7d1/index.js#L26-L30), and I assume that's the issue. Maybe there's another way.

**Comment by derhuerst** [comment]
Fixed by 6ddb583, published as `combine-tiles@1.0.0`. 🎉
